### PR TITLE
fixup records2table for new iterator pattern in SBP

### PR DESCRIPTION
**Pull request by denniszollo**
cc @mookerji

### CODE REVIEW

**Comment by mookerji** [comment]
Lovely.

File: gnss_analysis/tools/records2table.py [hunk 1]
```diff
@@ -325,11 +325,11 @@ def hdf5_write(log_datafile, filename, verbose=False):
   logging_interval = 10000
   start = time.time()
   with JSONLogIterator(log_datafile) as log:
-    for delta, timestamp, msg in log.next():
+    for msg, data in log.next():
       i += 1
       if verbose and i % logging_interval == 0:
         print "Processed %d records! @ %.1f sec." % (i, time.time() - start)
-      processor.process_message(delta, timestamp, msg)
+      processor.process_message(data.['delta'],  data.['timestamp'], msg)
```
Comment (mookerji) [inline gnss_analysis/tools/records2table.py]: uhhhhhhhh

Comment (mookerji) [inline gnss_analysis/tools/records2table.py]: big typo there...

**Comment by mookerji** [comment]
See comment. Don't merge yet.

**Comment by denniszollo** [comment]
Should be good now ... took me a while to even see the typo

**Comment by mookerji** [comment]
lol

**Comment by mookerji** [comment]
Thanks for fixing it!

**Pull request merged**